.travis.yml

```diff
@@ -16,10 +16,16 @@ sudo: false
 # versions 4-7 because the build for the first GK commit (that updates
 # package.json) will fail on those versions in the greenkeeper-lockfile-update
 # step.
-before_install: '[[ $(node -v) =~ ^v4.*$ ]] || [[ $(node -v) =~ ^v5.*$ ]] || [[ $(node -v) =~ ^v6.*$ ]] || [[ $(node -v) =~ ^v7.*$ ]] || npm install -g greenkeeper-lockfile'
+before_install:
+- '[[ $(node -v) =~ ^v4.*$ ]] || [[ $(node -v) =~ ^v5.*$ ]] || [[ $(node -v) =~ ^v6.*$ ]] || [[ $(node -v) =~ ^v7.*$ ]] || npm install -g greenkeeper-lockfile'
+- npm install --global @semantic-release/changelog @semantic-release/git
 install: npm install
 before_script: '[[ $(node -v) =~ ^v4.*$ ]] || [[ $(node -v) =~ ^v5.*$ ]] || [[ $(node -v) =~ ^v6.*$ ]] || [[ $(node -v) =~ ^v7.*$ ]] || greenkeeper-lockfile-update'
 after_script: '[[ $(node -v) =~ ^v4.*$ ]] || [[ $(node -v) =~ ^v5.*$ ]] || [[ $(node -v) =~ ^v6.*$ ]] || [[ $(node -v) =~ ^v7.*$ ]] || greenkeeper-lockfile-upload'
+deploy:
```

Comment:
this deploy step and semantic release both default to the `master` branch, so any testing we do around this config will need to be done there unless we jump through some hoops that might not be worth it. if this isnt something we want to do with `halfred`, i'm happy to move this to another project if there is a better suggestion.

Reply:
No, let's just do it with halfred's master branch, no need to complicate this.